azkaban-common/src/test/java/azkaban/utils/Md5HasherTest.java

```diff
@@ -0,0 +1,54 @@
+package azkaban.utils;
+
+import static org.junit.Assert.assertThat;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.LinkedList;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+import java.util.concurrent.Future;
+import java.util.concurrent.TimeUnit;
+import org.hamcrest.Matchers;
+import org.junit.Test;
+
+public class Md5HasherTest {
+
+  private static final File ZIP_FILE = new File("src/test/resources/sample_flow_01.zip");
+
+  @Test
+  public void md5Hash() {
+    assertMd5Hash();
+  }
+
+  @Test
+  public void md5HashParallel() throws Exception {
```

Comment:
why do we need test parallel scenario?

Reply:
> why do we need test parallel scenario?

Sorry, not needed. The possibility of a concurrency issue was just an idea as I was trying to find out why we have some hash differences.